scripts/generate-analysis-files-toolkit.sh

```diff
@@ -62,7 +62,8 @@ if [ "$RUN_LOCAL" -lt "1" ]; then
   cp ${analyses_dir}/focal-cn-file-preparation/results/consensus_wgs_plus_cnvkit_wxs.tsv.gz ${compiled_dir}
 
   # Move over the consensus with status file 
-  cp ${scratch_dir}/consensus_seg_with_status.tsv ${compiled_dir}
+  cp ${scratch_dir}/focal-cn-file-preparation/results/consensus_seg_with_status.tsv ${compiled_dir}
+  cp ${scratch_dir}/focal-cn-file-preparation/results/cnvkit_with_status.tsv ${compiled_dir}
```

Comment:
Can you remind me why we need CNVkit separate here?

Reply:
it is in the 03 script for TP53 module - see here: https://github.com/PediatricOpenTargets/OpenPedCan-analysis/pull/187/files.

Reply:
Oh, I see - for WXS only